Makefile

```diff
@@ -101,6 +101,9 @@ test:
 	@go test $(shell go list ./... | grep -v vendor)
 
 .PHONY: check-headers
-check-headers:
-	@./ci/headers-*.sh
+check-headers: ci/headers-*.sh
+
+.PHONY: ci/headers-*.sh
+ci/headers-*.sh:
+	@./$@
```

Comment:
I think just changing it to:

```
check-headers:
  ./ci/headers-bash.sh
  ./ci/headers-docker.sh
  ./ci/headers-go.sh
```

Would be best. Removing the `@` also makes it show which files fail in CI.

Reply:
I had it like that initially but then changed it into the more generic approach so that adding a new headers check would require no modifications to the Makefile. I find that a tad more convenient, WDYT?

I think the `@` is only suppressing the output of the line being executed, not the command's STDOUT itself. For instance, when I run the rule with a Dockerfile missing its header, I get the following:

```
$ make check-headers
license headers not found in ./cloud-controller-manager/cmd/digitalocean-cloud-controller-manager/Dockerfile
make: *** [ci/headers-docker.sh] Error 1
```

Reply:
I think clarity trumps convenience in this case, it would be better to clearly show what’s going on. Trying to be clever with convenience is what broke this in the first case :)

TIL what @ actually does in Make...

Reply:
I think clarity trumps convenience in this case, it would be better to clearly show what’s going on. Trying to be clever with convenience is what broke this in the first case :)

TIL what @ actually does in Make...

Reply:
@bouk convinced and updated. Extra benefit that the Makefile now doesn't stop on the first header type violation but executes them all. 👍 

PTAL again.